src/components/Comment.js

```diff
@@ -62,17 +65,29 @@ class Comment extends React.Component {
 
     if (editing) return <CommentForm commentId={comment.id} close={closeEdit} />
 
+    const { image } = comment
+
+    const encodeUrl = url => url.replace(/\//g, '%2F')
+
+    const showImage = isMobile
+      ? () => dispatch(navigate(`/image/${encodeUrl(image.url)}/${encodeUrl(location.pathname)}`))
+      : () => dispatch(showModal('image', {url: image.url}))
```

Comment:
I'd personally prefer:
```javascript
showImage = () => {
  if (isMobile) {
    dispatch(navigate(`/image/${encodeUrl(image.url)}/${encodeUrl(location.pathname)}`))
  } else {
    dispatch(showModal('image', {url: image.url}))
  }
}
```
... or similar. I'm just not a fan of the ternary operator for multi-line anythings.